packages/admin-panel/src/autocomplete/ReduxAutocomplete.jsx

```diff
@@ -67,6 +70,13 @@ const ReduxAutocompleteComponent = ({
     selectedValue = [];
   }
 
+  const renderOption = (option) => { 
```

Comment:
I think we could probably make `ReduxAutocomplete` accept a prop of `renderOption` and just pass that through instead, so that the `ReduxAutocomplete` component remains 'dumb'. 
So then `renderOption` here might look like the blow. (Note that I have renamed it because in this example `renderOption` would be a prop already and we can't have a function with the same name as a prop)

```
const getOptionRendered = (option) => {
   if(renderOption) return renderOption(option);
  if(!option || !option[optionLabelKey]) return '';
return option[optionLabelKey]
} 
```